src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/service/DisputeService.java

```diff
@@ -383,4 +383,28 @@ public DisputantUpdateRequest updateDisputantUpdateRequest(Long updateRequestId,
 		return disputantUpdateRequestRepository.save(disputantUpdateRequest);
 	}
 
+	/**
+	 * Updates the status of one or more DisputantUpdateRequest records to PENDING from HOLD only
+	 * @param updateRequestIds
+	 */
+	public void updateDisputantUpdateRequestsStatusPending(List<Long> updateRequestIds) {
+
+		List<DisputantUpdateRequest> disputantUpdateRequestList = disputantUpdateRequestRepository.findAllById(updateRequestIds);
```

Comment:
Leanne mentioned that there may be 3 or 30,000 update requests. Just thinking that maybe we should use the DisputeId to join to the DisputantUpdateRequest table to do a bulk query or update instead of passing all 30,000 ids in a query.

You can use the one disputeId to find the count(*) of DisputantUpdateRequests that are already on HOLD.

Reply:
Good point, I thought the same thing initially. However, these updates should be specific to each update request. If a user sends a request to update his address along with his email address and then changes his mind and send a second request to update his email again along with phone number, those two requests will be part of different sagas and transactions. If we use the disputeId to update the status of all the DisputantUpdateRequest records, that would cause a problem since it may update all the records which should be actually a part of different change process and saga. So I think we should keep the request specific to update records associated to a change request at a time. 

Also, that way there will be max of 3 ids in the update request since a user can request to update max of three different types (address, name, phone number) at once.